source/1_engine/rex_engine/include/rex_engine/defines.h

```diff
@@ -29,6 +31,19 @@
     #error DEBUG_BREAK unsupported machine instruction ...
   #endif
 
+//-------------------------------------------------------------------------
+// End of line
+namespace rex
```

Comment:
shouldn't really be in defines.h anymore now though..
but I'll make an issue for this so we can clean it up another time (I can take into my cleanup branch when I go over the repo)